distribution/lib/Standard/Base/0.0.0-dev/src/Network/Email.enso

```diff
@@ -0,0 +1,33 @@
+import project.Data.Text.Text
+import project.Data.Vector.Vector
+import project.Errors.Common.Missing_Argument
+import project.Errors.Common.Dry_Run_Operation
+import project.Network.Email.Email_Address.Email_Address
+import project.Network.Email.Email_Attachment.Email_Attachment
+import project.Network.Email.Email_Body.Email_Body
+import project.Network.Email.Email_Object.Email_Object
+import project.Network.Email.Email_Provider.Email_Provider
+import project.Network.Email.Email_Send_Response.Send_Response
+import project.Runtime.Context
+import project.Warning.Warning
+
+## ---
+   aliases: [mail]
+   group: Web
+   icon: email
+   ---
+   Sends an email.
+
+   Sends an email using the specified email provider. Only sends an email in write mode.
+   The TestOnly email provider can be used to simulate sending an email without actually sending it.
+@from Email_Address.default_widget
+@to Email_Address.vector_widget
+@cc Email_Address.vector_widget
+@bcc Email_Address.vector_widget
+@attachments Email_Attachment.vector_widget
+@body Email_Body.default_widget
+send from:Email_Address=(Missing_Argument.throw "from") to:(Vector Email_Address)=[] cc:(Vector Email_Address)=[] bcc:(Vector Email_Address)=[] subject:Text="" body:Email_Body="" attachments:(Vector Email_Attachment)=[] provider:Email_Provider=(Missing_Argument.throw "provider") -> Send_Response =
```

Comment:
we could merge Email module and type

- Move the `Email_Object` type into here and rename to `Email`.
  - Rename the constructor to `Message` (i.e. `Email.Message`)
  - Add a `new` constructor method
- Move the send static into the type. This stays as `Email.send` as this will be the main user method.
- Add a small member method `send_message` calling onto the static